python/py-pynetdicom3/Portfile

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+github.setup        pydicom pynetdicom3 7ebc172
+name                py-pynetdicom3
+version             1.0.0.dev0.20181211
```

Comment:
The version number is a bit odd. Keep in mind that sorting order when it becomes better later on might be an issue.

Reply:
Yeah, there's no official release of 1.0.0 yet, this (up through dev0) is their internal version string currently; maybe I'll just version to 1.0.0, and keep bumping the revision whenever I update the Portfile up to and including an actual release.

Reply:
Not sure what the right thing is, but I do believe if we do it the other way, "1.0.0.devwhatever" will end up appearing newer than "1.0.0" which won't be what we want.

Reply:
The latest tagged version is 0.9.1; I changed the version here to be 0.9.1-pre-1.0; hopefully this is clear enough that it is a prerelease, and this value should compare as less-than for any new tag of 0.9.2 or greater. How's that sound? Revision # can be bumped until there is an 'actual' release.